provider/source/src/tests/data.rs

```diff
@@ -690,6 +690,19 @@ impl SourceDataProvider {
                         ("Thai_graphclust_model4_heavy/weights.json", include_bytes!("../../tests/data/lstm/Thai_graphclust_model4_heavy/weights.json").as_slice())
                     ].into_iter().collect(),
                 )))),
+                tzdb_paths: Some(Arc::new(TzdbCache { root: AbstractFs::Memory(
+                    [
+                        ("tz-tag/africa", include_bytes!("../../tests/data/tzdb/africa").as_slice()),
```

Comment:
Question: where does `tz-tag` come from?

Reply:
If you download the zip from GitHub, there's a top-level directory with the tag name in it. The code in `source.rs` skips the top level directory, so I need one in testdata as well.